src/modules/form/components/group/DisabilityTable.tsx

```diff
@@ -141,18 +143,50 @@ const DisabilityTable = ({
   const statusId = useId();
   const disablingConditionId = useId();
   const disabilityTypeId = useId();
+  const isMobile = useIsMobile();
+
+  function getDisabilityDropdowns(
+    cellItem: FormItem,
+    idx: number,
+    disabilityTypeLabelId: string
+  ) {
+    return renderChildItem(cellItem, {
+      noLabel: true,
+      inputProps: {
+        label: null,
+        placeholder: idx === 0 ? 'Select Status' : 'Select Disabling Condition',
+        // Reads as "Physical Disability Status" or "Physical Disability Disabling Condition"
+        ariaLabelledBy: `${disabilityTypeLabelId} ${
```

Comment:
This shouldn't be included for mobile-view, if you take my suggestion above, since the inputs would be directly labeled.